consensus/state_sync/module.go

```diff
@@ -16,50 +15,22 @@ const (
 type StateSyncModule interface {
 	modules.Module
 	StateSyncServerModule
-	DebugStateSync
-
-	// This functions are used for managing the Server mode of the node, which is handled independently from the FSM.
-	IsServerModEnabled() bool
-	EnableServerMode() error
-	DisableServerMode() error
-
-	// This functions contains the business logic on handling Block and Metadata responses.
-	HandleStateSyncMetadataResponse(*typesCons.StateSyncMetadataResponse) error
-	HandleGetBlockResponse(*typesCons.GetBlockResponse) error
 
 	SendStateSyncMessage(msg *typesCons.StateSyncMessage, nodeAddress cryptoPocket.Address, height uint64) error
-
-	// Getter functions for the aggregated metadata and the metadata buffer, used by consensus module.
-	GetAggregatedMetadata() *typesCons.StateSyncMetadataResponse
-
-	// Starts syncing the node with the network by requesting blocks.
-	StartSyncing() error
-}
-
-// This interface should be only used for debugging purposes and tests.
-type DebugStateSync interface {
-	SetAggregatedMetadata(*typesCons.StateSyncMetadataResponse)
+	StateSyncLogHelper(receiverPeerAddress string) map[string]any
```

Comment:
A log helper should not be exposed publically. 

Have you ever seen a production interface in the standard go library called "logHelper"?